gio/src/subclass/action.rs

```diff
@@ -0,0 +1,254 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::{
+    ffi::{gboolean, GVariant, GVariantType},
+    prelude::*,
+    subclass::prelude::*,
+    translate::*,
+    GString, Variant, VariantType,
+};
+
+use crate::{ffi, Action};
+
+pub trait ActionImpl: ObjectImpl + ObjectSubclass<Type: IsA<Action>> {
+    fn name(&self) -> GString {
```

Comment:
Is it valid for this to change, or should this maybe just be an associated constant?

Reply:
Well, a name is a property of the action instance, not an action class.

Reply:
Indeed